mne/io/persyst/tests/test_persyst.py

```diff
@@ -207,6 +211,15 @@ def test_persyst_annotations(tmpdir):
 
     # make sure annotation with a "," character is in there
     assert 'seizure1,2' in annotations.description
+    assert 'CLip2' in annotations.description
+
+    # roundtripping annotations should work.
+    tmp_fpath = tmp_path / 'tmp_file.edf'
+    with pytest.warns(RuntimeWarning, match='EDF format'):
+        raw.export(tmp_fpath)
+    raw_edf = read_raw_edf(tmp_fpath)
+    for annot in raw.annotations:
+        assert annot['description'] in raw_edf.annotations.description
```

Comment:
remove this. Your assert above is enough. Don't test EDF export features here. Keept this in export module.

thx